### DISCUSSION TITLE
🔗 Show hide password in login form widget like in WordPress default (Duplicate of #21243)

### DISCUSSION THREAD

**Discussion by amikeliunas** [elementor/elementor#21264] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

I would like the Elementor PRO login form to show or hide the password content in login form like in the WordPress default

### Describe the solution you'd like

See clean-login version here https://www.ajbid.org/inicio-de-sesion/
Or default WP login https://www.ajbid.org/wp/wp-login.php
Both show EYE to show or hide password.
Elementor does NOT here https://www.ajbid.org/en/member-login/

I would like Elementor Pro to behave that way also with EYE icon as a toggle
Attached Files: https://my.elementor.com/wp-content/uploads/sites/2/gravity_forms/22-8fa82d82d4bd295a437b035bfb146a16/2023/02/system-info-www.ajbid_.org-03-02-2023.txt

### Describe alternatives you've considered

a fix for the Pro login widget [on/off toggle switch]

### Additional context

a snippet or custom Elementor code based on this https://stackoverflow.com/questions/69908009/placing-password-visibility-icon-inside-password-field

**Comment by nicholaszein** [comment]
Hello @amikeliunas,

thanks for submitting your feature request! 🙏 

Allow us to take some time to review your request. We will get back to you with feedback as soon as possible.

Best regards

**Comment by nicholaszein** [comment]
Hey @amikeliunas!

### 🔗 🚀 There is already a similar request submitted by someone here #21243

Feel free to continue the conversation and cast your **⬆️ Upvote** there! 👉

### 🔍 Please, in the future, remember to check if there isn't any thread similar to yours already opened. 

Kind regards
